packages/nimble-components/src/table-column/base/tests/table-column.fixtures.ts

```diff
@@ -58,3 +60,59 @@ export class TableColumnDelegatesClickAndKeydown extends TableColumn {
         });
     }
 }
+
+/**
+ * Column validator used by TableColumnValidationTest
+ */
+export class TestColumnValidator extends ColumnValidator<
+readonly ['invalidFoo', 'invalidBar']
+> {
+    public constructor(columnInternals: ColumnInternals<unknown>) {
+        super(columnInternals, ['invalidFoo', 'invalidBar'] as const);
+    }
+
+    public validateFoo(isValid: boolean): boolean {
+        return this.setConditionValue('invalidFoo', !isValid);
+    }
+
+    public validateBar(isValid: boolean): boolean {
+        return this.setConditionValue('invalidBar', !isValid);
+    }
+}
+
+export const tableColumnValidationTestTag = 'nimble-test-table-column-validation';
+/**
+ * Test column type used to verify column config validation
+ */
+@customElement({
+    name: tableColumnValidationTestTag
+})
+export class TableColumnValidationTest extends TableColumn {
+    @attr()
```

Comment:
`foo` and `bar` should be marked as `mode: 'boolean'`, right?

Reply:
Sure